pipelines/rj_smtr/constants.py

```diff
@@ -1198,6 +1198,14 @@ class constants(Enum):  # pylint: disable=c0103
         },
     }
 
+    GTFS_MATERIALIZACAO_PARAMS_NEW = {
```

Comment:
usando o flow antigo não precisa dessa constante, deixa pra modificar a constante antiga quando for descontinuar o dataset antigo do gtfs

Reply:
ok